### PR TITLE
Fixes #15253: Sending & Receiving files with the shared-files API

### PR DESCRIPTION
**Pull request by ArmandSy**
https://issues.rudder.io/issues/15253

### CODE REVIEW

**Comment by ArmandSy** [comment]
Commit modified

**Comment by ArmandSy** [comment]
Commit modified

**Comment by ArmandSy** [comment]
Commit modified

**Comment by ArmandSy** [comment]
Commit modified

**Comment by ArmandSy** [comment]
Commit modified

**Comment by amousset** [comment]
Can you rebase this PR with `rudder-dev rebase`?

**Comment by ArmandSy** [comment]
Commit modified

File: relay/sources/relayd/src/shared_files.rs [hunk 1]
```diff
@@ -39,12 +39,15 @@ use openssl::pkey::Public;
 use openssl::rsa::Rsa;
 use openssl::sign::Verifier;
 use regex::Regex;
+use reqwest;
```
Comment (amousset) [inline relay/sources/relayd/src/shared_files.rs]: This should be in the test module

**Pull request merged**